### ISSUE TITLE
Remove `agent_latest_pubkey` field from `agent_info()` or put behind `unstable-dpki` feature flag

### ISSUE THREAD

**Issue by mattyg** [holochain/holochain#4526]
This came from discussion around the developer documentation: https://github.com/holochain/docs-pages/pull/453#discussion_r1868211704

I propose we remove `agent_latest_pubkey` field from `agent_info()` or else put it behind `unstable-dpki` feature flag as it is an unimplemented feature, and has the same value as `agent_initial_pubkey`

**Comment by ThetaSinner** [comment]
Remove the field, if we need it back as part of DPKI integration it can be added back again

**Comment by matthme** [comment]
Should this also entail renaming `agent_initial_pubkey` to simply `agent_pubkey` then?

**Comment by jost-s** [comment]
I don't think so, because we'll likely bring back the latest pubkey field with DPKI.

**Comment by matthme** [comment]
Completed with #4815
